packages/ember-glimmer/lib/component.js

```diff
@@ -79,13 +78,16 @@ export const BOUNDS = symbol('BOUNDS');
   handle events or actions, you implement a subclass
   of `Ember.Component` named after the name of the
   component. Note that `Component` needs to be appended to the name of
-  your subclass like `AppProfileComponent`.
+  your subclass like `ProfileComponent`.
 
   For example, you could implement the action
-  `hello` for the `app-profile` component:
+  `hello` for the `profile` component:
 
-  ```javascript
-  App.AppProfileComponent = Ember.Component.extend({
+  ```app/components/profile-component.js
+
+  import Ember from 'ember';
+
+  ProfileComponent = Ember.Component.extend({
```

Comment:
As an example of a missing export, this could be rewritten `export default Ember.Component.extend({`.